rotkehlchen/db/utils.py

```diff
@@ -176,3 +178,16 @@ def is_valid_db_blockchain_account(
         return is_valid_polkadot_address(account)
 
     raise AssertionError(f'Unknown blockchain: {blockchain}')
+
+
+def should_save_balances(db: 'DBHandler') -> bool:
```

Comment:
Since the only argument is `DBHandler` why not make it a method in the `DBHandler` class?

Reply:
Yeahhhh. True. Will do that now